joomla/joomla-cms
# [4.0] Fix restore category/article from first version in version history

Pull Request for Issue #32890.

### Summary of Changes
Load data from current version before replacing it with data from history to avoid error if there are some required keys are missing in the history data. See https://github.com/joomla/joomla-cms/issues/32890 to understand more about the issue.


### Testing Instructions
1. See https://github.com/joomla/joomla-cms/issues/32890 , confirm the issue
2. Apply patch, confirm that the issue is sorted . Please test by both restore category and article from it's first version.
